src/wasm-s-parser.h

```diff
@@ -452,7 +450,6 @@ class SExpressionWasmBuilder {
   #define abort_on(str) { std::cerr << "aborting on " << str << '\n'; onError(); }
 
   Expression* parseExpression(Element& s) {
-    //if (debug) std::cerr << "parse expression " << s << '\n';
```

Comment:
just a cleanup. with the new pass times logging, logging is much more useful, and other excessive logging was drowning it out. removing it, i ended up removing all use of `debug` in this class, at which point the compiler errors on the property not being used, so I removed the property too, and at that stage, I figured leaving it in comments was silly.